pkg/webui/console/components/api-key-modal/index.js

```diff
@@ -78,9 +78,14 @@ const ApiKeyModal = function(props) {
 }
 
 ApiKeyModal.propTypes = {
-  visible: PropTypes.bool.isRequired,
+  rights: PropTypes.rights,
   secret: PropTypes.string,
```

Comment:
```suggestion
  secret: PropTypes.string.isRequired,
```
From my understanding these should be required, but double check if that leads to problems.

Reply:
`secret` and `rights` are `undefined` here, so this results in a PropType error